### PR TITLE
Persisting winners to the database and have setting the winner not ef…

### PR DESCRIPTION
**Pull request by markcatley**
…fect the result of participate calls if the alternative is already selected.

### CODE REVIEW

**Comment by harmoney-justins** [comment]
should we be creating a PR in the main project as well?

File: sixpack/models.py [hunk 1]
```diff
@@ -424,7 +430,7 @@ def find_or_create(cls, experiment_name, alternatives,
             experiment.set_traffic_fraction(traffic_fraction)
             experiment.save()
 
-        # Only check traffic fraction if the experiment is being updated 
+        # Only check traffic fraction if the experiment is being updated
```
Comment (nicole-ashley) [inline sixpack/models.py:433]: Unnecessary change

**Comment by markcatley** [comment]
They're unlikely to accept this PR as it changes the base functionality. Keen to open an issue in the main project once we're happy with this to see how the functionality could be integrated if they are interested.

File: sixpack/models.py (hunk 1 above)
Comment (markcatley) [inline sixpack/models.py:433]: /shrugs :-P

**Comment by nicole-ashley** [comment]
👍 Looks good to me, small tweaks but they have a huge benefit in a funnels where you need to lock down the alternative no matter where it came from.

File: sixpack/models.py (hunk 1 above)
Comment (nicole-ashley) [inline sixpack/models.py:433]: Never mind, I notice it was removed rather than added.
